cmd/kpod/main.go

```diff
@@ -15,6 +15,7 @@ func main() {
 	if reexec.Init() {
 		return
 	}
+	logrus.SetLevel(logrus.ErrorLevel)
```

Comment:
pretty sure this is redundant, isn't logrus defaulting to, at least, error level?